### DISCUSSION TITLE
`yarn test` fails with fresh fork and clone

### DISCUSSION THREAD

**Discussion by cgrindel** [renovatebot/renovate#21343] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

Expected tests to be green.

### What you tried so far.

Verified that I installed the minimum version for the tools listed in the [local-development doc](https://github.com/renovatebot/renovate/blob/main/docs/development/local-development.md).

I searched the repository looking for similar issues. I found [this issue](https://github.com/renovatebot/renovate/issues/19999), which reports a similar error. It is unclear to me if the resolution in the issue addresses this error.

### Relevant debug logs

<details><summary>Logs</summary>

```
 FAIL  lib/util/github/graphql/cache-strategies/package-cache-strategy.spec.ts (104 MB heap size)
  ● util/github/graphql/cache-strategies/package-cache-strategy › reconciles old cache record with new items

    expect(received).toEqual(expected) // deep equality

    - Expected  - 1
    + Received  + 1

    @@ -21,8 +21,8 @@
                "releaseTimestamp": "2022-10-15T18:00:00.000-06:00",
                "version": "4",
              },
            },
          },
    -     21600,
    +     21660,
        ],
      ]

      53 |     expect(res).toEqual([item1, item2, updatedItem, newItem]);
      54 |     expect(isPaginationDone).toBe(true);
    > 55 |     expect(cacheSet.mock.calls).toEqual([
         |                                 ^
      56 |       [
      57 |         'foo',
      58 |         'bar',

      at Object.<anonymous> (lib/util/github/graphql/cache-strategies/package-cache-strategy.spec.ts:55:33)

 PASS  lib/workers/repository/config-migration/branch/migrated-data.spec.ts (35.971 s, 839 MB heap size)
```

</details>

**Comment by rarkins** [comment]
Could be a timezone problem? You have a value 60 different than expected, but I haven't checked if that's seconds or minutes

I just cloned a fresh copy to `/tmp` on a MacBook with Node 18, ran `yarn && yarn install` and got a fully successful result

**Reply by cgrindel** [reply]
Once I get up to speed on the docs, I will investigate to see if it is a timezone issue. We should be able to update the test handle it, if it is a timezone discrepancy.

**Reply by rarkins** [reply]
Yes, it shouldn't happen and I'm surprised it's only happening to you. We have contributors in quite a few time zones

**Reply by cgrindel** [reply]
Agreed. However, I did find [someone else report this error](https://github.com/renovatebot/renovate/issues/19999). The error message is very similar.

**Comment by cgrindel** [comment, marked as answer]
I have submitted [a fix for this issue](https://github.com/renovatebot/renovate/pull/21389).
